<launch>
    <node name="stmotion_controller_bringup1" pkg="stmotion_controller" type="stmotion_controller_bringup" output="screen" required="true">
        <param name="config_fname" value="$(find stmotion_controller)/config/user_config.json"/>
        <param name="robot_name" value="r1"/>
        <param name="root_pwd" value="$(find stmotion_controller)/"/>
    </node>
</launch>